src/token/stream-parser.ts

```diff
@@ -256,26 +255,22 @@ class Parser {
     });
   }
 
-  readBigInt64LE(callback: (data: JSBI) => void) {
+  readBigInt64LE(callback: (data: BigInt) => void) {
     this.awaitData(8, () => {
-      const result = JSBI.add(
-        JSBI.leftShift(
-          JSBI.BigInt(
-            this.buffer[this.position + 4] +
-            this.buffer[this.position + 5] * 2 ** 8 +
-            this.buffer[this.position + 6] * 2 ** 16 +
-            (this.buffer[this.position + 7] << 24) // Overflow
-          ),
-          JSBI.BigInt(32)
-        ),
-        JSBI.BigInt(
+      const result =
+        (BigInt(
+          this.buffer[this.position + 4] +
+          this.buffer[this.position + 5] * 2 ** 8 +
+          this.buffer[this.position + 6] * 2 ** 16 +
+          (this.buffer[this.position + 7] << 24) // Overflow
+        ) << BigInt(32)
+        ) +
+        BigInt(
           this.buffer[this.position] +
           this.buffer[this.position + 1] * 2 ** 8 +
           this.buffer[this.position + 2] * 2 ** 16 +
           this.buffer[this.position + 3] * 2 ** 24
-        )
-      );
-
+        );
```

Comment:
Instead of doing the calculations ourselves here, I think we can just call out to [`readBigInt64LE`](https://nodejs.org/docs/latest-v14.x/api/buffer.html#buffer_buf_readbigint64le_offset).

Reply:
got it, I will do a fallowing up commit for this. Thanks for clear this up!